### ISSUE TITLE
[BUG] Bottleneck is not reported correctly with newer versions of Veeam

### ISSUE THREAD

**Issue by tigattack** [tigattack/VeeamNotify#19]
Bottleneck is always undefined at https://github.com/tigattack/VeeamNotify/blob/dev/AlertSender.ps1#L105-L113

Veeam broke this somewhere between 11.0.0.837 and latest.

https://forums.veeam.com/powershell-f26/accessing-bottleneck-info-via-veeam-backup-model-cbottleneckinfo-bottleneck-t80127.html

**Comment by tigattack** [comment]
PoC workaround

```powershell
# Get latest session ID
Get-VBRBackupSession | sort EndTime | select -last 1 -Property id
# 'Extract' logger method
$vbrSessionLogger = (Get-VBRBackupSession | Where-Object {$_.Id.Guid -eq 'session ID here'}).Logger
# Get bottleneck
(($vbrSessionLogger.GetLog() | Select-Object -ExpandProperty UpdatedRecords | Where-Object {$_.Title -match 'Primary bottleneck:.*'} | Select-Object -ExpandProperty Title) -replace 'Primary bottleneck:','').Trim()
```

**Comment by tigattack** [comment]
Alternative workaround from a [comment](https://forums.veeam.com/post449290.html#p449290) on the forum post linked in the original issue.

Manually 
```ps1
$source = $session.Progress.BottleneckInfo.Source
$proxy = $session.Progress.BottleneckInfo.Proxy
$network = $session.Progress.BottleneckInfo.Network
$target = $session.Progress.BottleneckInfo.Target
$maxSourceTarget = [Math]::Max($source, $target)
$maxProxyNetwork = [Math]::Max($proxy, $network)
$maxFinal = [Math]::Max($maxSourceTarget, $maxProxyNetwork)
switch ($maxFinal) {
    $source {
        $bottleneck = 'Source'
    }
    $proxy {
        $bottleneck = 'Proxy'
    }
    $network {
        $bottleneck = 'Network'
    }
    $target {
        $bottleneck = 'Target'
    }
}
```

**Comment by tigattack** [comment]
Resolved by #20
